.storybook/config.js

```diff
@@ -33,3 +38,18 @@ addParameters({
 
 configure(loadStories, module);
 injectGlobal(...fonts);
+
+function setCurrentStory(story, variation) {
+  clearCurrentStory();
```

Comment:
If this isn't here, loading the same story does not reset. The Storybook APIs will only clear if a story changes.